examples/build.rs

```diff
@@ -182,6 +182,8 @@ fn main() -> anyhow::Result<()> {
     let robot_name = cloud_cfg.local_fqdn.split('.').next().unwrap_or("");
     let local_fqdn = cloud_cfg.local_fqdn.replace('.', "-");
     let fqdn = cloud_cfg.fqdn.replace('.', "-");
+    // let location_secret = cloud_cfg.location_secrets.first().unwrap_or("");
```

Comment:
why the commented lines?

Reply:
webhook related. was trying to get the location secret that we decided to get via board attributes for now instead.